services/storage/tests/test_dsm.py

```diff
@@ -364,3 +366,116 @@ def test_fmd_build():
+          "template-uuid-48eb-a9d2-aaad6b72400a": {
+            "key": "simcore/services/frontend/file-picker",
+            "version": "1.0.0",
+            "label": "File Picker",
+            "inputs": {},
+            "inputNodes": [],
+            "outputNode": False,
+            "outputs": {
+              "outFile": {
+                "store": 1,
+                "path": "Shared Data/Height-Weight.csv"
+              }
+            },
+            "progress": 100,
+            "thumbnail": "",
+            "position": {
+              "x": 100,
+              "y": 100
+            }
+          },
+          "template-uuid-4c63-a705-03a2c339646c": {
+            "key": "simcore/services/dynamic/raw-graphs",
+            "version": "2.8.0",
+            "label": "2D plot",
+            "inputs": {
+              "input_1": {
+                "nodeUuid": "template-uuid-48eb-a9d2-aaad6b72400a",
+                "output": "outFile"
+              }
+            },
+            "inputNodes": [
+              "template-uuid-48eb-a9d2-aaad6b72400a"
+            ],
+            "outputNode": False,
+            "outputs": {},
+            "progress": 0,
+            "thumbnail": "",
+            "position": {
+              "x": 400,
+              "y": 100
+            }
+          }
+        }
+    }
+
+    bucket_name = BUCKET_NAME
+    s3_client.create_bucket(bucket_name, delete_contents_if_exists=True)
+
+    source_project["workbench"]["template-uuid-48eb-a9d2-aaad6b72400a"]["outputs"]["outFile"]["path"] = path_in_datcore
+
+    destination_project = copy.deepcopy(source_project)
+    source_project_id = source_project["uuid"]
+    destination_project["uuid"] = source_project_id.replace("template", "deep-copy")
+    destination_project["workbench"] = {}
+
+    node_mapping = {}
+
+    for node_id, node in source_project["workbench"].items():
+        object_name = str(Path(source_project_id) / Path(node_id) / Path(node_id + ".dat"))
+        f = utils.data_dir() / Path("notebooks.zip")
+        s3_client.upload_file(bucket_name, object_name, f)
+        key = node_id.replace("template", "deep-copy")
+        destination_project["workbench"][key] = node
+        node_mapping[node_id] = key
+
+    status = await dsm.deep_copy_project_simcore_s3(user_id, source_project, destination_project, node_mapping)
+    new_path = destination_project["workbench"]["deep-copy-uuid-48eb-a9d2-aaad6b72400a"]["outputs"]["outFile"]["path"]
+    assert new_path != path_in_datcore
+    files = await dsm.list_files(user_id=user_id, location=SIMCORE_S3_STR)
+    assert len(files) == 3
+    found = False
+    for f in files:
+        if f.file_uuid == new_path:
+            found = True
+
+    assert found
+
+    response = await dsm.delete_project_simcore_s3(user_id, destination_project["uuid"])
+    print(response)
+
+    files = await dsm.list_files(user_id=user_id, location=SIMCORE_S3_STR)
+    assert len(files) == 0
+
+async def test_mock(mocker, dsm_fixture):
```

Comment:
I think you can remove this test, right? it is a dev test so I would mark as 

```python
 @pytest.mark.skip(reason-"dev-test")  
```

Reply:
true that